lib/svg.dart

```diff
@@ -16,6 +16,13 @@ import 'src/svg/theme.dart';
 import 'src/utilities/file.dart';
 import 'src/vector_drawable.dart';
 
+/// Builder function to create an error widget.
+typedef LoadingErrorWidgetBuilder = Widget Function(
+    BuildContext context,
+    dynamic error,
```

Comment:
Please avoid dynamic.

```suggestion
    Object error,
```